backend/inspirehep/legacy/views.py

```diff
@@ -115,3 +115,19 @@ def redirect_collection(legacy_collection):
 @blueprint.route("/info/<path:info_path>", methods=("GET",))
 def redirect_info(info_path):
     return redirect(f"{current_app.config['LEGACY_BASE_URL']}/info/{info_path}", 302)
+
+
+@blueprint.route("/legacy/arxiv/<arxiv>", methods=("GET",))
```

Comment:
why `/legacy` in the route?

Reply:
cause we will change the route in ingress, otherwise we were not able to keep the route with `/api/orcid/<orcid>` without redirect